packages/react/package.json

```diff
@@ -11,7 +11,7 @@
     "@stardust-ui/react-component-ref": "^0.38.0",
     "@stardust-ui/react-proptypes": "^0.38.0",
     "classnames": "^2.2.5",
-    "downshift": "^3.2.10",
+    "downshift": "^3.2.14",
```

Comment:
Heads up, because we use the caret `^`, consumers' `yarn install @stardust-ui/react` will automatically resolve to the latest `3.x.x` version.

If we want consumers to get `3.2.x` then we need to specify the tilde `~3.2.10`, which will only allow the latest patch version.

Lastly, for users using yarn, the `yarn.lock` file will remember whatever actual version was resolved for a given `package.json` specified version.  So, there is that caveat.  Example, if a user installed Stardust when downshift `3.2.14` was available as the latest `3.x.x` version, then the resolution of `^3.2.10` would point to `3.2.14` and they would get that version every time until they updated their `yarn.lock`.

Reply:
sorry for the rollup issue, it wasn't obvious for me.

I will update to 3.2.14 fixed version in this PR. 3.3.0 is released but I did not ask for approval on that one.

Is this ok?